libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -118,22 +193,81 @@ namespace pika::mpi::experimental {
             std::array<mpi_stream, max_mpi_streams> default_queues_;
         };
 
+        struct initializer
+        {
+            initializer()
+            {
+                init_throttling_default();
+            }
+        };
+
```

Comment:
What does this do? If there's no destructor you could just call `init_throttling_default` directly? But this also looks unused.

Reply:
left over from experiments with throttling. Would like to leave it for now and remove it or use it properly when I re-enable throttling